<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SitesDialog</class>
 <widget class="QDialog" name="SitesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>. . .</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_selectfile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Select an .xyz file</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_atop">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Atop</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_site_type</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_bridge">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Bridge</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_site_type</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3hollow">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>3-way Hollow</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_site_type</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4hollow">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>4-fold Hollow</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_site_type</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_selectsite">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Select which sites you wish to analyze :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_analyze">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_shape">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_shape">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Select (best approximate) shape: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_atomicZ">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Insert Atomic number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_atomicZ">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>78</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_all">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_site_type</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename_out">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_outputfile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Please select the folder where you wish any output file(s) to be stored.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fileout">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>. . .</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_site_type"/>
 </buttongroups>
</ui>
